### PR TITLE
Add Dashboard Navigation Test

### PR DESCRIPTION
**Pull request by engineSound**
New Dashboard Navigtation Testing.
Dashboard is read only page. It doesn't have write operation on the system. The part that is important to test is the "Show get started" link. The PR reflects the test.

@ndebuhr The dev-hs-5 is still in the configuration.  I think i should remove it in the PR. What do you think?

### CODE REVIEW

**Comment by ndebuhr** [comment]
@engineSound Should be live on beta pretty soon.  Maybe let's use the beta url?

File: hydroshare/hydroshare.py [hunk 1]
```diff
@@ -713,6 +713,22 @@ def oracle():
...
+        Dashboard.toggle_get_started(self.driver)
+        Home.to_home(self.driver)
+        oracle()
+        
+
 
 if __name__ == '__main__':
     parse_args_run_tests(HydroshareTestSuite)
+
+
```
Comment (ndebuhr) [inline hydroshare/hydroshare.py]: Let's remove the extra lines at the end of the file here.

File: hydroshare/hs_elements.py [hunk 2]
```diff
@@ -502,7 +503,11 @@ def resource_creation_type(self, index):
             '#dropdown-resource-type ul li:nth-of-type({})'.format(index),
         )
 
+class DashboardPage:
+    def __init__(self):
+        self.get_started_toggle = SiteElement(By.ID, 'change_me')
```
Comment (ndebuhr) [inline hydroshare/hs_elements.py]: The 'change_me' ID works, but I'd recommend somehow noting this as a TODO in hydroshare - using a non-placeholder ID.  Something like id="get_started_toggle". If you can change that in the source code quickly, then we can change the test before merging the PR.  Whatever you think is best.

File: hydroshare/hydroshare.py (hunk 1 above)
Comment (engineSound) [reply hydroshare/hydroshare.py]: good idea. Beta will have the dashboard in a  few days.

File: hydroshare/hs_elements.py (hunk 2 above)
Comment (engineSound) [reply hydroshare/hs_elements.py]: yes. I will plan to once the dashboard pr is approved.

File: hydroshare/hydroshare.py (hunk 1 above)
Comment (engineSound) [reply hydroshare/hydroshare.py]: beta.hydroshare.org

Comment (engineSound) [reply hydroshare/hydroshare.py]: changed to beta.

File: hydroshare/hs_elements.py (hunk 2 above)
Comment (engineSound) [reply hydroshare/hs_elements.py]: updated id="id-getting-started-toggle" 
When beta.hydroshare.org is online, you will see the new id.

**Pull request closed without merging**